evasquez26/angular4-startup
# Registration - change text

## Story / Task

see [ONE-226](https://bixlabs.atlassian.net/browse/ONE-226)


## What this PR does?

Fixed bug. Change text in sign up modal.


### Before 

To sign up with social networks, the text was: "Or sign in with" 

### After

![screenshot from 2017-05-25 11-20-23](https://cloud.githubusercontent.com/assets/20344883/26460474/5b5fbb70-413f-11e7-8afe-af5dcafa3815.png)


### How to test / reproduce

- Start the app.
- Press the "Sign up" button.
- On the last part of the emerging pop-up, you should be able to read "Or sign up with"

### TODOs
N/A

### Related PRs
N/A

## Migrations
N/A


------------
Thanks!
